resources/istio/templates/istio-proxy-reset.yaml

```diff
@@ -64,10 +64,16 @@ spec:
         - name: proxy-reset
           image: eu.gcr.io/kyma-project/tpi/k8s-tools:20210407-86c769bd
           env:
-          - name: EXPECTED_ISTIO_PROXY_IMAGE
-            value: {{ .Values.kyma.proxyResetJob.commonIstioProxyImagePrefix }}:{{ .Chart.Version }}
-          - name: COMMON_ISTIO_PROXY_IMAGE_PREFIX
+          - name: ISTIO_PROXY_IMAGE_PREFIX
             value: {{ .Values.kyma.proxyResetJob.commonIstioProxyImagePrefix }}
+          - name: ISTIO_PROXY_IMAGE_VERSION
+            value: {{ .Chart.Version }}
+          - name: RETRIES_COUNT
+            value: "5"
+          - name: DRY_RUN
+            value: "false"
+          - name: EXIT_CODE
+            value: "0"
```

Comment:
Shouldn't this be exposed as {{ .Values... }} so it can be easily configured? Changing it now would require manual operations on a cluster or changing Kyma version (e.g. from 1.21 to master)